vm-setup/roles/v1aX_integration_test/templates/cluster-template-controlplane.yaml

```diff
@@ -1,23 +1,20 @@
 kind: KubeadmControlPlane
-apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
+apiVersion: controlplane.cluster.x-k8s.io/{{ CAPI_VERSION }}
 metadata:
   name: ${ CLUSTER_NAME }
   namespace: ${ NAMESPACE }
 spec:
   rolloutStrategy:
     rollingUpdate:
       maxSurge: ${ MAX_SURGE_VALUE }
```

Comment:
According to this https://github.com/kubernetes-sigs/cluster-api/blob/master/docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md#percentage-string-or-int-api-input-will-fail-with-a-string-different-from-an-integer-with--appended, maxSurge will need a '%' from now on to be a valid input. Are we handling this anywhere?

Reply:
I think they mean that it must either be a string with a "%" or an integer (so not a string). I.e. `"3%"` and `3` is fine, but `"3"` is not. Good question though! And especially when grabbing the value from an environment variable.

Reply:
I think we are using the integer value and this change is needed if we use string.